Runtime/BlockadeLabsSkyboxGenerator.cs

```diff
@@ -504,9 +504,13 @@ private async Task<GetImagineResult> PollForResultAsync(string imagineObfuscated
 
         private Task WaitForSeconds(float seconds)
         {
-            var tcs = new TaskCompletionSource<object>();
-            StartCoroutine(WaitForSecondsEnumerator(tcs, seconds));
-            return tcs.Task;
+            #if UNITY_EDITOR
+                return Task.Delay((int)(seconds * 1000));
```

Comment:
```suggestion
                return Task.Delay((int)(seconds * 1000)).ConfigureAwait(true);
```

You could also use editor coroutines and use EditorWaitForSecondsRealtime